Index: ../src/com/company/Main.java
===================================================================
diff --git a/../src/com/company/Main.java b/../src/com/company/Main.java
deleted file mode 100644
--- a/../src/com/company/Main.java	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,46 +0,0 @@
-package com.company;
-import java.util.Scanner;
-
-public class Main {
-    public static void main(String[] args) {
-        Scanner scanner = new Scanner(System.in);
-        int height, length, width;
-        String color, material;
-        String[][] array = new String[5][3];
-
-        for (int counter = 0; counter < 5; counter++){
-            System.out.print("введите размеры " + (counter + 1) + "-го куба:" +
-                    "\nвысота: ");
-            height = scanner.nextInt();
-            System.out.print("длина: ");
-            length = scanner.nextInt();
-            System.out.print("ширина: ");
-            width = scanner.nextInt();
-            Box box = new Box(height, length, width);
-            array[counter][0] = String.valueOf(box.calculateVolume());
-            System.out.print("цвет: ");
-            color = scanner.next();
-            box.setColor(color);
-            array[counter][1] = box.getColor();
-            System.out.print("материал: ");
-            material = scanner.next();
-            box.setMaterial(material);
-            array[counter][2] = box.getMaterial();
-        }
-
-        System.out.println();
-
-        for (int count = 0; count < 5; count++) {
-            System.out.println("объем куба " + (count + 1) + "-го куба: " + array[count][0] +
-                    ";\tцвет: " + array[count][1] +
-                    "; материал: " + array[count][2]);
-        }
-
-        System.out.println("ЕСЛИ ЗНАЧЕНИЕ ПОЛЯ ОБЪЕМА КУБА РАВНО 0 ЭТО ЗНАЧИТ, ЧТО ВЫ ВВЕЛИ НЕКОРРЕКТНОЕ ЗНАЧЕНИЕ");
-    }
-}
-
-/*
-для комментариев и замечания...
-=)
-*/
\ No newline at end of file
Index: ../src/com/company/Box.java
===================================================================
diff --git a/../src/com/company/Box.java b/../src/com/company/Box.java
deleted file mode 100644
--- a/../src/com/company/Box.java	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,74 +0,0 @@
-package com.company;
-import java.util.Scanner;
-
-public class Box {
-    private Scanner s = new Scanner(System.in);
-    private int height, length, width;
-    private String color, material;
-
-    public Box(){}
-
-    public Box(int height, int length, int width){
-        if (height > 0 && length > 0 && width > 0){
-            this.length = length;
-            this.height = height;
-            this.width = width;
-        }
-    }
-
-    public Box(int height, int length, int width, String color, String material){
-        new Box(height, length, width);
-        this.color = color;
-        this.material = material;
-    }
-
-    public void setHeight(int height) {
-        if (height > 0){
-            this.height = height;
-        }
-    }
-
-    public void setMaterial(String material) {
-        this.material = material;
-    }
-
-    public void setColor(String color) {
-        this.color = color;
-    }
-
-    public void setWidth(int width) {
-        if (width > 0){
-            this.width = width;
-        }
-    }
-
-    public void setLength(int length) {
-        if (length > 0){
-            this.length = length;
-        }
-    }
-
-    public String getMaterial() {
-        return material;
-    }
-
-    public String getColor() {
-        return color;
-    }
-
-    public int getWidth() {
-        return width;
-    }
-
-    public int getLength() {
-        return length;
-    }
-
-    public int getHeight() {
-        return height;
-    }
-
-    public int calculateVolume(){
-        return height*length*width;
-    }
-}
\ No newline at end of file
Index: ../.gitignore
===================================================================
diff --git a/../.gitignore b/../.gitignore
deleted file mode 100644
--- a/../.gitignore	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,2 +0,0 @@
-# Project exclude paths
-/out/
\ No newline at end of file
Index: ../.idea/.gitignore
===================================================================
diff --git a/../.idea/.gitignore b/../.idea/.gitignore
deleted file mode 100644
--- a/../.idea/.gitignore	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: ../.idea/encodings.xml
===================================================================
diff --git a/../.idea/encodings.xml b/../.idea/encodings.xml
deleted file mode 100644
--- a/../.idea/encodings.xml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="Encoding">
-    <file url="PROJECT" charset="UTF-8" />
-  </component>
-</project>
\ No newline at end of file
Index: ../projects.iml
===================================================================
diff --git a/../projects.iml b/../projects.iml
deleted file mode 100644
--- a/../projects.iml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: ../.idea/project-template.xml
===================================================================
diff --git a/../.idea/project-template.xml b/../.idea/project-template.xml
deleted file mode 100644
--- a/../.idea/project-template.xml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,3 +0,0 @@
-<template>
-  <input-field default="com.company">IJ_BASE_PACKAGE</input-field>
-</template>
\ No newline at end of file
Index: ../.idea/description.html
===================================================================
diff --git a/../.idea/description.html b/../.idea/description.html
deleted file mode 100644
--- a/../.idea/description.html	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ /dev/null	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
@@ -1,1 +0,0 @@
-<html>Simple <b>Java</b> application that includes a class with <code>main()</code> method</html>
\ No newline at end of file
Index: ../idea/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/vcs.xml b/../idea/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1615734326151)
+++ b/../idea/.idea/vcs.xml	(date 1615734326151)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: ../idea/src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/src/com/company/Main.java b/../idea/src/com/company/Main.java
new file mode 100644
--- /dev/null	(date 1615734395276)
+++ b/../idea/src/com/company/Main.java	(date 1615734395276)
@@ -0,0 +1,7 @@
+package com.company;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+    }
+}
\ No newline at end of file
Index: ../idea/src/com/company/Box.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/src/com/company/Box.java b/../idea/src/com/company/Box.java
new file mode 100644
--- /dev/null	(date 1615499008964)
+++ b/../idea/src/com/company/Box.java	(date 1615499008964)
@@ -0,0 +1,97 @@
+package com.company;
+import java.util.Scanner;
+
+public class Box {
+    private  int height, length, width;
+    private static String color, material;
+    private Scanner scanner = new Scanner(System.in);
+
+    public Box(){}
+
+    public Box(int height, int length, int width){  //высота, длина, ширина
+        if (length > 0 && height > 0 && width > 0){
+            this.height = height;
+            this.width = width;
+            this.length = length;
+        }else{
+            this.height = 0;
+            this.width = 0;
+            this.length = 0;
+
+        }
+    }
+
+    public Box(int height, int length, int width, String color, String material){
+        new Box(height, length, width);
+        this.color = color;
+        this.material = material;
+    }
+
+    public void setHeight(int height) {
+        if (height > 0){
+            this.height = height;
+        }
+    }
+
+    public void setWidth(int width) {
+        if (width > 0){
+            this.width = width;
+        } else {
+            this.width = 0;
+        }
+    }
+
+    public void setLength(int length) {
+        if (length > 0){
+            this.length = length;
+        } else {
+            this.length = 0;
+        }
+    }
+
+    public String setMaterial(String material) {
+        this.material = material;
+        return material;
+    }
+
+    public String setColor(String color) {
+        this.color = color;
+        return this.color;
+    }
+
+    public int getHeight(){
+        if (height <= 0 ){
+            return Integer.parseInt(null);
+        } else {
+            return height;
+        }
+    }
+
+    public int getLength() {
+        if (length <= 0){
+            return Integer.parseInt(null);
+        } else {
+            return length;
+        }
+    }
+
+    public int getWidth() {
+        if (width <= 0){
+            return Integer.parseInt(null);
+        } else {
+            return width;
+        }
+    }
+
+    public static String getColor() {
+        return color;
+    }
+
+    public static String getMaterial() {
+        return material;
+    }
+
+    public int calculateVolume(){
+        return height*length*width;
+    }
+}
\ No newline at end of file
Index: ../enums/enums.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/enums.iml b/../enums/enums.iml
new file mode 100644
--- /dev/null	(date 1615734463231)
+++ b/../enums/enums.iml	(date 1615734463231)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: ../enums/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/misc.xml b/../enums/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1615734463408)
+++ b/../enums/.idea/misc.xml	(date 1615734463408)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectKey">
+    <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
+  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: ../enums/.idea/project-template.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/project-template.xml b/../enums/.idea/project-template.xml
new file mode 100644
--- /dev/null	(date 1615734393619)
+++ b/../enums/.idea/project-template.xml	(date 1615734393619)
@@ -0,0 +1,3 @@
+<template>
+  <input-field default="com.company">IJ_BASE_PACKAGE</input-field>
+</template>
\ No newline at end of file
Index: ../enums/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/modules.xml b/../enums/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1615734463331)
+++ b/../enums/.idea/modules.xml	(date 1615734463331)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/enums.iml" filepath="$PROJECT_DIR$/enums.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: ../enums/src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/src/com/company/Main.java b/../enums/src/com/company/Main.java
new file mode 100644
--- /dev/null	(date 1615766313697)
+++ b/../enums/src/com/company/Main.java	(date 1615766313697)
@@ -0,0 +1,20 @@
+package com.company;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Car car;
+        System.out.print("цвет: ");
+        String color = scanner.nextLine();
+        if (color.equals(Color.BLUE.getDescription())){
+            car = new Car(Color.BLUE);
+        } else if (color.equals(Color.RED.getDescription())){
+            car = new Car(Color.RED);
+        } else {
+            car = new Car(Color.BLACK);
+        }
+        System.out.println(car.getColor());
+        System.out.println(car.getColor().getDescription());
+    }
+}
Index: ../enums/.idea/description.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/description.html b/../enums/.idea/description.html
new file mode 100644
--- /dev/null	(date 1615734393299)
+++ b/../enums/.idea/description.html	(date 1615734393299)
@@ -0,0 +1,1 @@
+<html>Simple <b>Java</b> application that includes a class with <code>main()</code> method</html>
\ No newline at end of file
Index: ../enums/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/.gitignore b/../enums/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1615734411983)
+++ b/../enums/.idea/.gitignore	(date 1615734411983)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: ../enums/src/com/company/Color.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/src/com/company/Color.java b/../enums/src/com/company/Color.java
new file mode 100644
--- /dev/null	(date 1615735270074)
+++ b/../enums/src/com/company/Color.java	(date 1615735270074)
@@ -0,0 +1,17 @@
+package com.company;
+
+public enum Color {
+    RED("красный"),
+    BLUE("синий"),
+    BLACK("черный");
+
+    private String description;
+
+    Color(String description) {
+        this.description = description;
+    }
+
+    public String getDescription(){
+        return description;
+    }
+}
Index: ../enums/src/com/company/Car.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/src/com/company/Car.java b/../enums/src/com/company/Car.java
new file mode 100644
--- /dev/null	(date 1615765861282)
+++ b/../enums/src/com/company/Car.java	(date 1615765861282)
@@ -0,0 +1,12 @@
+package com.company;
+
+public class Car {
+    private Color color;
+
+    public Car(Color color){
+        this.color = color;
+    }
+    public Color getColor(){
+        return color;
+    }
+}
Index: ../enums/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/vcs.xml b/../enums/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1615734463444)
+++ b/../enums/.idea/vcs.xml	(date 1615734463444)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: ../enums/.idea/encodings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../enums/.idea/encodings.xml b/../enums/.idea/encodings.xml
new file mode 100644
--- /dev/null	(date 1615734463474)
+++ b/../enums/.idea/encodings.xml	(date 1615734463474)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Encoding">
+    <file url="PROJECT" charset="UTF-8" />
+  </component>
+</project>
\ No newline at end of file
Index: ../idea/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/modules.xml b/../idea/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1615227813963)
+++ b/../idea/.idea/modules.xml	(date 1615227813963)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/idea.iml" filepath="$PROJECT_DIR$/idea.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: ../idea/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/misc.xml b/../idea/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1615227814061)
+++ b/../idea/.idea/misc.xml	(date 1615227814061)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectKey">
+    <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
+  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: ../idea/.idea/encodings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/encodings.xml b/../idea/.idea/encodings.xml
new file mode 100644
--- /dev/null	(date 1615227814107)
+++ b/../idea/.idea/encodings.xml	(date 1615227814107)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Encoding">
+    <file url="PROJECT" charset="UTF-8" />
+  </component>
+</project>
\ No newline at end of file
Index: ../idea/.idea/description.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/description.html b/../idea/.idea/description.html
new file mode 100644
--- /dev/null	(date 1615227697338)
+++ b/../idea/.idea/description.html	(date 1615227697338)
@@ -0,0 +1,1 @@
+<html>Simple <b>Java</b> application that includes a class with <code>main()</code> method</html>
\ No newline at end of file
Index: ../idea/.idea/project-template.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/project-template.xml b/../idea/.idea/project-template.xml
new file mode 100644
--- /dev/null	(date 1615227697408)
+++ b/../idea/.idea/project-template.xml	(date 1615227697408)
@@ -0,0 +1,3 @@
+<template>
+  <input-field default="com.company">IJ_BASE_PACKAGE</input-field>
+</template>
\ No newline at end of file
Index: ../idea/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/.idea/.gitignore b/../idea/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1615227728136)
+++ b/../idea/.idea/.gitignore	(date 1615227728136)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: ../.idea/projects.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/projects.iml b/../.idea/projects.iml
new file mode 100644
--- /dev/null	(date 1615729514117)
+++ b/../.idea/projects.iml	(date 1615729514117)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: ../.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/projects.iml\" filepath=\"$PROJECT_DIR$/projects.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/modules.xml b/../.idea/modules.xml
--- a/../.idea/modules.xml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ b/../.idea/modules.xml	(date 1615729564385)
@@ -2,7 +2,9 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/projects.iml" filepath="$PROJECT_DIR$/projects.iml" />
+      <module fileurl="file://$PROJECT_DIR$/idea/idea.iml" filepath="$PROJECT_DIR$/idea/idea.iml" />
+      <module fileurl="file://$PROJECT_DIR$/lesson11/lesson11.iml" filepath="$PROJECT_DIR$/lesson11/lesson11.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/projects.iml" filepath="$PROJECT_DIR$/.idea/projects.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
Index: ../.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"VcsDirectoryMappings\">\r\n    <mapping directory=\"$PROJECT_DIR$\" vcs=\"Git\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/vcs.xml b/../.idea/vcs.xml
--- a/../.idea/vcs.xml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ b/../.idea/vcs.xml	(date 1615729514316)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="" vcs="Git" />
   </component>
 </project>
\ No newline at end of file
Index: ../idea/idea.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../idea/idea.iml b/../idea/idea.iml
new file mode 100644
--- /dev/null	(date 1615729564362)
+++ b/../idea/idea.iml	(date 1615729564362)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: ../.idea/workspace.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/workspace.xml b/../.idea/workspace.xml
new file mode 100644
--- /dev/null	(date 1615729564416)
+++ b/../.idea/workspace.xml	(date 1615729564416)
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ChangeListManager">
+    <list default="true" id="1083ac50-f754-447a-8a52-b93cee526163" name="Default Changelist" comment="">
+      <change beforePath="$PROJECT_DIR$/.gitignore" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/.gitignore" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/description.html" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/encodings.xml" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/misc.xml" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/modules.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/modules.xml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/project-template.xml" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/.idea/vcs.xml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/vcs.xml" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/projects.iml" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/com/company/Box.java" beforeDir="false" />
+      <change beforePath="$PROJECT_DIR$/src/com/company/Main.java" beforeDir="false" />
+    </list>
+    <option name="SHOW_DIALOG" value="false" />
+    <option name="HIGHLIGHT_CONFLICTS" value="true" />
+    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
+    <option name="LAST_RESOLUTION" value="IGNORE" />
+  </component>
+  <component name="Git.Settings">
+    <option name="RECENT_GIT_ROOT_PATH" value="$PROJECT_DIR$" />
+  </component>
+  <component name="ProjectId" id="1pkaHWaj0SveVV7pa86fO2xYd7P" />
+  <component name="ProjectViewState">
+    <option name="hideEmptyMiddlePackages" value="true" />
+    <option name="showLibraryContents" value="true" />
+  </component>
+  <component name="PropertiesComponent">
+    <property name="RunOnceActivity.OpenProjectViewOnStart" value="true" />
+    <property name="RunOnceActivity.ShowReadmeOnStart" value="true" />
+    <property name="last_opened_file_path" value="$PROJECT_DIR$/lesson11" />
+  </component>
+  <component name="SpellCheckerSettings" RuntimeDictionaries="0" Folders="0" CustomDictionaries="0" DefaultDictionary="application-level" UseSingleDictionary="true" transferred="true" />
+  <component name="TaskManager">
+    <task active="true" id="Default" summary="Default task">
+      <changelist id="1083ac50-f754-447a-8a52-b93cee526163" name="Default Changelist" comment="" />
+      <created>1615729480755</created>
+      <option name="number" value="Default" />
+      <option name="presentableId" value="Default" />
+      <updated>1615729480755</updated>
+    </task>
+    <servers />
+  </component>
+</project>
\ No newline at end of file
Index: ../.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"EntryPointsManager\">\r\n    <entry_points version=\"2.0\" />\r\n  </component>\r\n  <component name=\"ProjectKey\">\r\n    <option name=\"state\" value=\"project://e2804f05-5315-4fc6-a121-c522a6c26470\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_15\" default=\"true\" project-jdk-name=\"15\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/misc.xml b/../.idea/misc.xml
--- a/../.idea/misc.xml	(revision 4d1576f712cf40350deba3a66f08b2be87e3bbda)
+++ b/../.idea/misc.xml	(date 1615729564462)
@@ -1,12 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="EntryPointsManager">
-    <entry_points version="2.0" />
-  </component>
-  <component name="ProjectKey">
-    <option name="state" value="project://e2804f05-5315-4fc6-a121-c522a6c26470" />
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" project-jdk-name="15" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
